contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -41,7 +41,7 @@ specifically, a test is eligible for promotion to conformance if:
   to pre-pull images for conformance tests)
 - it works without non-standard filesystem permissions granted to pods
 - it does not rely on any binaries that would not be required for the linux
-  kernel or kubelet to run (e.g., can't rely on git)
+  kernel or kubelet to run (e.g., can't rely on git), or differ in behavior based on OS (nslookup, ping)
```

Comment:
Make this a separate bullet point instead of adding to this one.

Reply:
moved to separate bullet